modules/sor/lib/poolsV3/basePoolV3.ts

```diff
@@ -13,7 +13,7 @@ export class BasePoolV3 {
     public readonly chain: Chain;
     public readonly id: Hex;
     public readonly address: string;
-    public declare readonly poolType: PoolType;
+    public declare readonly poolType: PoolType | string;
```

Comment:
Should have this strongly typed as PoolType if possible. Seems to be needed in the SDK?

Reply:
I usually prefer strong typed, but here is seems like an unnecessary friction.
Having it as `PoolType` requires us to update the SDK and bump its dependency within backend repo, which is actually not needed.
One of the main benefits from the new SDK is that is supports new v3 pool types without having to manually add them to the SDK itself.
Any specific reason why we should make it more strict?